portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +30,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function toggle (section) {
+  const container = document.getElementById(section);
+  const other = document.getElementById(section === 'swimming' ? 'rowing': 'swimming');
+  if (container.classList.contains('invisible')) {
+    container.classList.remove('invisible')
+    other.classList.add('invisible');
+  } else {
+    container.classList.add('invisible')
```

Comment:
nit: since we're reusing 'invisible' a lot, let's put it into a variable. This way, if the 'invisible' id changes to something else, or you want to add a different id, you only need to change the variable once and not the string literals four times.

In general, it's good practice to put all string literals and constants into variables, even if they're only used once (since we might not know when we'll need to reuse them).